### PR TITLE
feat: move attendee to waitlist

### PR DESCRIPTION
**Pull request by gikf**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

Ref #2330

---
- Adds moving attendee (confirmed) to the waitlist.

### CODE REVIEW

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Looks great! I only spotted a couple of minor things:

File: cypress/e2e/dashboard/events/[eventId].cy.ts [hunk 1]
```diff
@@ -122,6 +157,7 @@ describe('event dashboard', () => {
         cy.login(users.testUser.email);
 
         cy.deleteAttendee(eventId, confirmedUser.id).then(expectToBeRejected);
+        //cy.moveToWaitlist(eventId, confirmedUser.id).then(expectToBeRejected);
```
[the remark this answers is not in the document]
Comment (gikf) [reply cypress/e2e/dashboard/events/[eventId].cy.ts:160]: Oh, right. Good I added this placeholder comment, to not forget about adding cypress command to test if mutation is rejected for user :eyes:

[the remark this answers is not in the document]
Comment (ojeytonwilliams) [reply cypress/e2e/dashboard/events/[eventId].cy.ts:160]: Hahah, awesome. Mission accomplished, then!

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Pull request merged**